packages/editor/src/components/post-preview-button/index.js

```diff
@@ -89,26 +89,136 @@ export class PostPreviewButton extends Component {
 		this.props.autosave();
 
 		const markup = `
-			<div class="editor-post-preview-button__interstitial-message">
-				<p>Please wait&hellip;</p>
-				<p>Generating preview.</p>
+			<div class="post">
+  				<h1 class="editor-post-title__input"></h1>
```

Comment:
Using multiple empty H1s is semantically not-so-great.

I think it'd make more sense to have this kind of markup:

```html
<h1>Generating Preview</h1>
<img class="cover-image" alt="Placeholder animation for cover image." />
<p>Please wait for preview…</p>
<span class="fake-paragraph"></span>
<span class="fake-header"></span>
<span class="fake-paragraph"></span>
<span class="fake-image"></span>
<span class="fake-list"></span>

[ etc. ]
```

You'll need to create semantically unimportant elements that *look* like paragraphs, lists, etc. or this content will be quite confusing to screenreaders.

Reply:
This isn't resolved, but the markup seems to have changed so GitHub thinks it is.